README.md

```diff
@@ -1,70 +1,75 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# CRYTPO-MARKET
+In this project I built a metrics webapp using the CoinGeckoApi. This application displays a list of ranking cryptocurrencies
+and allows you to search in the home page any of your favorite crypto coins. When users click one cryptocurrency they will be able to see things like CoinMarketCap rank, Current value, Symbol, price changes from 60d to 1h, and more.
 
-## Available Scripts
+ ## Learning objectives
+- Use React components.
+- Use React props.
+- Use React state.
+- Use Redux
+- Use Events in a React App
+- Use React Hooks
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- JavaScript
+- React
+- Redux
+- Jest
+- Gitflow
+- Linters Check
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Website Preview
+![Preview](./src/images/Screenshot%202022-11-22%20at%204.19.31%20PM.png)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Live Demo
+Please use this [link](https://effervescent-stroopwafel-cbea62.netlify.app)
```

Comment:
- [ ] your app works as expected locally however in the deployed version I can't any cards so please make sure that deployed version also working as expected 
- Yours 
<img width="405" alt="Screenshot 2022-11-22 at 17 40 49" src="https://user-images.githubusercontent.com/61790587/203372166-5b58b28d-1f62-4e92-961e-de7e8a3d993e.png">

- Screenshot of the requirements 
<img width="1062" alt="Screenshot 2022-11-22 at 17 44 11" src="https://user-images.githubusercontent.com/61790587/203372095-ea8e43d1-0116-4411-a988-ad3c901101e1.png">

Reply:
Hi @youmari . I have tested it with my cell phone and other devices and the link works. I suppose it is probably due to network issues when you first open it.

Reply:
Hi @Jaymelfah your deployed app works properly 👍🏿